cmdutil/service/standard.go

```diff
@@ -74,6 +74,9 @@ func New(appConfig interface{}, ofs ...OptionFunc) *Standard {
 	}
 
 	s.Add(debug.New(logger, sc.Debug.Port))
+	if sc.Debug.EnablePprofDebugging {
+		s.Add(debug.NewPProfServer(logger, sc.Debug.PprofPort, sc.Debug.MutexProfileFraction))
```

Comment:
You could pass these in on a config struct that is embedded on the `debub.Config`:
```
type PprofConfig {
// fields
}
```
and then call `debug.NewPProfServer(sc.Debug.Pprof)` to set up the server.